app/models/moment.rb

```diff
@@ -42,4 +42,8 @@ def array_data
     end
   end
 
+  def strategy
```

Comment:
because all other classes, except for strategy has a method that is it's name in singular, I added this alias method to strategy.